src/combinatorics/binomial-coefficients.md

```diff
@@ -96,3 +96,8 @@ If the entire table of values is not necessary, storing only two last rows of it
 ### Calculation in $O(1)$
 
 Finally, in some situations it is beneficial to pre-calculate all the factorials in order to produce any necessary binomial coefficient with only two divisions later.This can be advantageous when using [long arithmetic](./algebra/big-integer.html), when the memory does not allow precalculation of all Pascal's triangle, or when you need to do calculations for some prime modulo (if the modulo is not prime, there are difficulties when dividing the numerator by the denominator, which can be overcome by factoring modulo and storing all numbers in the form of vectors of powers of these primes; see the section [Long arithmetic in factored form](./algebra/big-integer.html)).
+
+## Practice Problems
+ - [HACKEREARTH: Binomial Coefficient](https://www.hackerearth.com/problem/algorithm/binomial-coefficient-1/description/)
+ - [HACKEREARTH: Different sums](https://www.hackerearth.com/practice/math/combinatorics/basics-of-combinatorics/practice-problems/approximate/different-sums/)
```

Comment:
I'm not sure Different Sums and Power Set Game are good problems to practice binomial coefficients on. They are in "Basics of Combinatorics" tutorial but they are not actually about binomial coefficients. We probably want something more straightforward for practice section.